paasta_tools/cli/schemas/tron_schema.json

```diff
@@ -445,7 +445,7 @@
                             "type": "boolean"
                         },
                         "spark.app.name": {
-                            "type": "boolean"
+                            "type": "string"
```

Comment:
nice-to-have suggestion: might be worth adding some additional constraints here too re: min/max length since (iirc) this gets turned into a k8s label and those values have a max length (and if there's any characters that are not allowed, there's a field for adding a pattern that these strings need to match to pass validation)